uaclient/cli.py

```diff
@@ -80,7 +80,7 @@ def new_f(args, cfg):
 
 def attach_parser(parser=None):
     """Build or extend an arg parser for attach subcommand."""
-    usage = USAGE_TMPL.format(name=NAME, command="attach [token]")
+    usage = USAGE_TMPL.format(name=NAME, command="attach <token>")
```

Comment:
This is a total aside, but what's the history behind us setting our own usage message rather than using the default?

Reply:
historically I had tried to get the argparse output to look a bit more like lxd cli output (as was a initial ux request)